onnxruntime/contrib_ops/cpu/activations.h

```diff
@@ -28,5 +29,19 @@ class ScaledTanh final : public OpKernel {
   const float beta_;
 };
 
+template <typename T>
+class Gelu : public OpKernel {
+ public:
+  Gelu(const OpKernelInfo& info) : OpKernel(info) {}
+
+  Status Compute(OpKernelContext* context) const override {
+    const auto* X = context->Input<Tensor>(0);
+    Tensor* Y = context->Output(0, X->Shape());
+    EIGEN_X_VAR(xm);
+    EIGEN_Y = xm * 0.5f * ((xm * static_cast<float>(M_SQRT1_2)).erf() + 1.0f);
+    return Status::OK();
```

Comment:
Should this use MlasComputeErf?

Reply:
I actually don't know how to call  MlasComputeErf(X.template Data<float>(), Y.template MutableData<float>(), X.Shape().Size()); within this function (which is using EIGEN). 

Actually I don't know whether we could use the kernel output buffer (e.g. Y)  in place, could you please share some ideas?  If I want to use MlasComputeErf, then we need calculate "((xm * static_cast<float>(M_SQRT1_2))" leveraing EIGEN and store the results in Y. Then we call MlasComputeErf with Y as input, and also Output (otherwise we will need another buffer called Z). The last step, use EIGEN to compute the left computation.

Reply:
OK. We can revisit if it ever shows up as a perf issue.